cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -459,4 +474,9 @@ export class Application {
                 });
         });
     }
+    applicationInformation(): void {
+        this.applicationDetailsTab(details);
```

Comment:
Rename this method to validateApplicationInfo() or if you are just clicking on pencil Icon here then rename it to 

editApplicationFromApplicationProfile()

Reply:
Done, Thanks !